app/scripts/migrations/052.js

```diff
@@ -0,0 +1,24 @@
+import { cloneDeep } from 'lodash';
+
+import { removeStorageItem } from '../../../ui/lib/storage-helpers';
+
+const version = 52;
+
+/**
+ * Removes storage key no longer required due to move to
+ * fetchWithCache for gas retrieval and storage
+ */
+export default {
+  version,
+  async migrate(originalVersionedData) {
+    const versionedData = cloneDeep(originalVersionedData);
+    versionedData.meta.version = version;
+
+    await Promise.all([
+      await removeStorageItem('BASIC_PRICE_ESTIMATES_LAST_RETRIEVED'),
+      await removeStorageItem('BASIC_PRICE_ESTIMATES')
```

Comment:
I'm guessing this is what you meant to do? Calling `await` directly on each of these will make them execute serially, which defeats the purpose of using `Promise.all`

```suggestion
      removeStorageItem('BASIC_PRICE_ESTIMATES_LAST_RETRIEVED'),
      removeStorageItem('BASIC_PRICE_ESTIMATES')
```

Reply:
I'm curious if this will also fix the benchmark/metrics timeout

Reply:
/me facepalms.  My bad, not sure how I didn't catch that.